backend/spec/requests/api/v1/project_developers_spec.rb

```diff
@@ -38,6 +38,91 @@
         end
       end
     end
+
+    post "Create new Project Developer" do
+      tags "Project Developers"
+      consumes "multipart/form-data"
```

Comment:
Hmm, the plan was to use DirectUpload to not use multipart/form-data. Of course, that will be more work on the frontend. Maybe we could do it in another iteration to not focus on details now. We have to set file size limits, and also validate if for example images are actually images (I think active storage analyze using imagemagick or vips is doing that).

Reply:
I have not thought about direct upload approach, but it is good option. For now I will keep `multipart/form-data`, but I have also added `direct-upload` to rswag documentation. There are not extra changes required if we use direct-upload or multipart/form-data approach now. Later on, we can get rid of multipart/form-data and switch to pure json again if required.

About validation issue, I have created https://vizzuality.atlassian.net/browse/LET-226 to address this, because this is another quite complex topic so lets keep it independent.

Reply:
yes, let's keep validation separate 👍